lib/services/platform-service.ts

```diff
@@ -416,10 +417,13 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		await this.trackActionForPlatform({ action: "Build", platform, isForDevice });
 
 		let platformData = this.$platformsData.getPlatformData(platform, projectData);
-		platformData.platformProjectService.on(constants.BUILD_OUTPUT_EVENT_NAME, (data: any) => {
+		const handler = (data: any) => {
 			this.emit(constants.BUILD_OUTPUT_EVENT_NAME, data);
-		});
-		await platformData.platformProjectService.buildProject(platformData.projectRoot, projectData, buildConfig);
+			this.$logger.printInfoMessageOnSameLine(data.data.toString());
```

Comment:
why do we have this line and we do no have it in the other cases when we use attachAwaitDetach?